ui/apps/platform/src/Containers/Vulnerabilities/WorkloadCves/Deployment/DeploymentPageVulnerabilities.tsx

```diff
@@ -253,6 +259,10 @@ function DeploymentPageVulnerabilities({
                                 setPage(1);
                                 trackAppliedFilter(WORKLOAD_CVE_FILTER_APPLIED, searchPayload);
                             }}
+                            additionalContextFilter={{
+                                ...baseSearchFilter,
+                                'Deployment ID': deploymentId,
+                            }}
```

Comment:
Adds the base search filter to the autocomplete context, and fixes a bug across all three detail pages where the results are not scoped to the current CVE/Image/Deployment.